iroh-gateway/src/bad_bits.rs

```diff
@@ -191,9 +191,13 @@ mod tests {
 
         let rpc_addr = "grpc://0.0.0.0:0".parse().unwrap();
         let content_loader = RpcClient::new(config.rpc_client.clone()).await.unwrap();
+        tokio::spawn(async move {
+            if let Err(err) = crate::rpc::new(rpc_addr, crate::rpc::Gateway::default()).await {
```

Comment:
Slightly unsure how this actually communicates with the GW itself. I guess there's no commands that it uses that need to talk to GW yet.